muspinsim_config/muspinsim_config.xml

```diff
@@ -69,248 +71,166 @@
                         <option value="dissipation">dissipation</option>
                     </param>
                     <when value="zeeman">
-                        <param name="zeeman_index" type="integer" value="" label="Index of coupled spin" min="1"
-                               help="Index refers to the order of the spins listed in 'spins' section.  The first spin in the list has index 1, the second has index 2, and so on."/>
-                        <param name="zeeman_vector" type="text" value="" label="Zeeman coupling vector"
-                            help="Define 1X3 vector for local magnetic field coupling (T).
-                            Allows default expressions, constants and functions (see help)"/>
+                        <param name="zeeman_index" type="integer" value="" label="Index of coupled spin" min="1" help="Index refers to the order of the spins listed in 'spins' section.  The first spin in the list has index 1, the second has index 2, and so on."/>
+                        <param name="zeeman_vector" type="text" value="" label="Zeeman coupling vector" help="Define 1X3 vector for local magnetic field coupling (T). Allows default expressions, constants and functions (see help)"/>
```

Comment:
Maybe here should a validator be added? I think there was already one in some other tools that checked a matrix like input.

Reply:
Thanks for the suggestion. I think it’s more complicated than its worth here since this input should not only take in digits separated by spaces but also variables and basic python expressions.